frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -7,6 +7,7 @@ local Event = require("ui/event")
 local Geom = require("ui/geometry")
 local InfoMessage = require("ui/widget/infomessage")
 local InputDialog = require("ui/widget/inputdialog")
+local IsoLanguage = require("ui/isolanguage")
 local JSON = require("json")
```

Comment:
You'd better require() it only in the function where you need it. We don't want all these language name strings to say in memory each time KOreader is launched - but only for the rare session this menu will be used.